schemas/v3.1/schema.yaml

```
@@ -0,0 +1,809 @@
+                      - allOf:
+                          - $ref: '#CookieParam'
+                          - $ref: '#/$defs/Mixins/$defs/StyledFormOnly'
+              - allOf:
+                  - required:
+                      - content
+                  - $ref: '#/$defs/Mixins/$defs/WithContent'
+                  - $ref: '#/$defs/Mixins/$defs/WithEmpty'
+            unevaluatedProperties: false
+        $defs:
+          PathParam:
+            $anchor: PathParam
+            properties:
+              in:
+                const: path
+          QueryParam:
+            $anchor: QueryParam
+            properties:
+              in:
+                const: query
+          HeaderParam:
+            $anchor: HeaderParam
+            properties:
+              in:
+                const: header
+          CookieParam:
+            $anchor: CookieParam
+            properties:
+              in:
+                const: cookie
+      RequestBody:
+        $anchor: RequestBody
+        required:
+          - content
+        allOf:
+          - $ref: '#/$defs/Mixins/$defs/Describable'
+          - $ref: '#/$defs/Mixins/$defs/Requireable'
+          - $ref: '#/$defs/Mixins/$defs/Extensible'
+        properties:
+          content:
+            type: object
+            additionalProperties:
+              $ref: '#MediaType'
+        unevaluatedProperties: false
+      SecurityScheme:
+        $anchor: SecurityScheme
+        oneOf:
+          - $ref: '#Reference'
+          - type: object
+            required:
+              - type
+            allOf:
+              - $ref: '#/$defs/Mixins/$defs/Describable'
+              - $ref: '#/$defs/Mixins/$defs/Extensible'
+            properties:
+              type:
+                string: null
+                enum:
+                  - apiKey
+                  - http
+                  - bearer
+                  - mutualTLS
+                  - oauth2
+                  - openIdConnect
+                oneOf:
+                  - $ref: '#APIKey'
+                  - $ref: '#HTTP'
+                  - $ref: '#MutualTLS'
+                  - $ref: '#OAuth2'
+                  - $ref: '#OpenIdConnect'
+            unevaluatedProperties: false
+        $defs:
+          APIKey:
+            $anchor: APIKey
+            properties:
+              type:
+                const: apiKey
+            allOf:
+              - $ref: '#/$defs/Mixins/$defs/WithNameAndLocation'
+            not:
+              $ref: '#PathParam'
+          HTTP:
+            $anchor: HTTP
+            required:
+              - scheme
+            properties:
+              scheme:
+                type: string
+            oneOf:
+              - properties:
+                  type:
+                    const: http
+              - properties:
+                  type:
+                    const: bearer
+                  bearerFormat:
+                    type: string
```

This part doesn't look right to me. The logic is supposed to be
> if `scheme` (not `type`!) = `"bearer"`
> then `bearerFormat` may be present

https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.1.0.md#fixed-fields-23

Something like:

```suggestion
            properties:
              type:
                const: http
              scheme:
                type: string
            if:
              properties:
                scheme:
                  const: bearer
            then:
              properties:
                bearerFormat:
                  type: string
```